app/test_main.py

```diff
@@ -1,3 +1,19 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+def test_incorrect_year_format() -> None:
+    with pytest.raises(ValueError):
+        get_human_age(-12, 43.1)
+
+
+def test_with_zero_expected() -> None:
+    assert get_human_age(12, 14) == [0, 0]
```

Comment:
The expected result `[0, 0]` for `get_human_age(12, 14)` seems incorrect based on the conversion logic. According to the conversion rules, this should likely be `[1, 1]`. Please verify and update the expected result accordingly.